src/service/service.ts

```diff
@@ -1,6 +1,6 @@
 import { EventEmitter } from "events";
 import debug from "debug";
-import Multiaddr = require("multiaddr");
+import { Multiaddr } from "multiaddr";
 import isIp = require("is-ip");
 import PeerId = require("peer-id");
```

Comment:
PeerId can be imported with `import` already right?

Reply:
done, also updated in other places